app/heron/factories/plate.rb

```diff
@@ -38,7 +38,9 @@ def save # rubocop:todo Metrics/MethodLength
         ActiveRecord::Base.transaction do
           @plate = purpose.create!
 
-          Barcode.create!(asset: @plate, barcode: barcode, format: barcode_format)
+          # Overriding labware=() function causes issues in the factory
```

Comment:
What am I meant to take from this comment? It reads like you had a problem that perhaps you have resolved now?

Reply:
I _had_ a problem, and I solved it. Forgot to remove the comment 😄 I'll remove it.